style.css

```diff
@@ -1,7 +1,45 @@
+hr {
+  height: 3px !important;
+  width: 200px;
+  background-color: #6e5cc2 !important;
+  margin: 20px auto;
+  opacity: 1 !important;
+}
+
+form .btn:hover {
+  background-color: #0d6efd;
+  color: white;
+}
+
+/* Start List */
+.books-container {
+  max-height: 300px;
+  overflow-y: scroll;
+}
```

Comment:
- [ ] While you free to use styles of your choice (colors, fonts, backgrounds, etc.), you are required to follow the general layout as much as possible. Please see below differences between your page presentation and the wireframe template.

Your design: Please specify the book is by the author as provided in the wireframe design. It makes a good user experience. You don't want your users getting confused :smile: 

![Screenshot 2021-09-16 at 09-34-17 Awesome Books](https://user-images.githubusercontent.com/70150423/133588661-8e0badcc-d6cf-4dd2-b77d-4e7071c6fe7f.png)

Wireframe template.
![Screenshot_2021-09-16_09-38-26](https://user-images.githubusercontent.com/70150423/133589402-24a42111-388a-4b59-827c-6cb278cdc7b5.png)